pkg/apis/extensions/validation/validation_test.go

```diff
@@ -1282,14 +1282,15 @@ func TestValidateDeploymentStatus(t *testing.T) {
 			observedGeneration: 1,
 			expectedErr:        true,
 		},
+		/* TODO: Enable when we stop supporting updates from 1.5.
```

Comment:
I'd just remove this unit test. We'll add a unit test when we add that validation back. We may have more fields in the future and this test would need to be updated anyways.